_src/pldoc/static/js/ui.js

```diff
@@ -1,142 +1,111 @@
-    }
-
-    // palette values
-    function rgbaToHex(rgb) {
-        if (typeof(rgb) !== 'undefined') {
-            var _rgb = rgb.match(/^rgba?[\s+]?\([\s+]?(\d+)[\s+]?,[\s+]?(\d+)[\s+]?,[\s+]?(\d+)[\s+]?/i),
-                hex;
-
-            hex = (_rgb && _rgb.length === 4) ? '#' +
-                ('0' + parseInt(_rgb[1],10).toString(16)).slice(-2) +
-                ('0' + parseInt(_rgb[2],10).toString(16)).slice(-2) +
-                ('0' + parseInt(_rgb[3],10).toString(16)).slice(-2) : '';
-        } else {
-            hex = ' --- ';
-        }
+                hex = (_rgb && _rgb.length === 4) ? '#' +
+                    ('0' + parseInt(_rgb[1],10).toString(16)).slice(-2) +
+                    ('0' + parseInt(_rgb[2],10).toString(16)).slice(-2) +
+                    ('0' + parseInt(_rgb[3],10).toString(16)).slice(-2) : '';
+            }
+
+            return hex;
+        },
 
-        return hex;
-    }
-
-    if ($('.example').length) {
-
-        $('.example').each(function() {
-            var rgb = $(this).css('backgroundColor');
-
-            $(this).find('.color-meta .color-rgb').text(rgb);
-            $(this).find('.color-meta .color-hex').text(rgbaToHex(rgb));
-        });
-    }
-
-    // this selects all the text in an element
-    // http://stackoverflow.com/questions/12243898/how-to-select-all-text-in-contenteditable-div
-    $.fn.selectText = function() {
-        var doc = document,
-            element = this[0];
-
-        if (doc.body.createTextRange) {
-            var range = doc.body.createTextRange();
-            range.moveToElementText(element);
-            range.select();
-        } else if (window.getSelection) {
-            var selection = window.getSelection();
-            var range = doc.createRange();
-            range.selectNodeContents(element);
-            selection.removeAllRanges();
-            selection.addRange(range);
+        selectText: function(that) {
+            // this selects all the text in an element
+            // http://stackoverflow.com/questions/12243898/how-to-select-all-text-in-contenteditable-div
+            var doc = document,
+                element = that[0];
+
+            if (doc.body.createTextRange) {
+                var range = doc.body.createTextRange();
+                range.moveToElementText(element);
+                range.select();
+            } else if (window.getSelection) {
+                var selection = window.getSelection();
+                var range = doc.createRange();
+                range.selectNodeContents(element);
+                selection.removeAllRanges();
+                selection.addRange(range);
+            }
+        },
+
+        listenForPaletteClick: function() {
+            $('.is-copyable').on('click', function() {
+                Ui.selectText($(this));
+            });
         }
-    }
+    };
 
-    $('.is-copyable').click(function() {
-        $(this).selectText();
-    });
-});
+    Ui.init();
+});
```

Comment:
Oh, I thought you were talking about the specific line and was confused. Okay, on it.